### ISSUE TITLE
Pipes reset when switch to recent apps

### ISSUE THREAD

**Issue by ItzSelenux** [VadimBoev/FlappyBird#24]
As the title says, pipes resets when switching to the recent apps. It also happens without battery saver

OS: Android 10 AOSP
Device: Redmi Note 8
Build: 1.5 from GitHub releases

https://github.com/user-attachments/assets/24fefa29-887b-493d-b0e0-53873865c62a

**Comment by VadimBoev** [comment]
Yes. This is due to the complete lack of processing of these events. Along the way, over time, I think such issues will be resolved

**Comment by VadimBoev** [comment]
fixed in https://github.com/VadimBoev/FlappyBird/commit/b240a2b4690b516b62ae5ad4bc1ff113dd0cb126
